test/e2e/manifest.yaml

```diff
@@ -11,11 +11,11 @@ DriverInfo:
     exec: true
     multipods: true
     RWX: true
-    # TODO: Re-enable fsGroup tests: https://bugzilla.redhat.com/show_bug.cgi?id=2019128
+    # FSGroup is not supported in this release
     fsGroup: false
```

Comment:
Why is fsGroup not supported? I tested it last week for azurefile, it was working fine with mount options.

Reply:
Fixed

Reply:
From https://issues.redhat.com/browse/STOR-760:
```
The feature did not get GA in Kubernetes 1.24. It's beta there, enabled by default, and we will support it as GA for Azure File CSi driver.
```